shared/src/business/useCaseHelper/pendingItems/fetchPendingItemsByDocketNumber.js

```diff
@@ -31,7 +31,7 @@ exports.fetchPendingItemsByDocketNumber = async ({
   let foundDocuments = [];
 
   docketEntries.forEach(document => {
-    if (document.pending && document.servedAt) {
+    if (document.pending && (document.servedAt || document.isLegacyServed)) {
```

Comment:
This is also similar to the code added in Case.js - Maybe move the logic out to a reusable function.

Reply:
Refactoring card added https://trello.com/c/BQLzXAuD